### ISSUE TITLE
PutNextLocal

### ISSUE THREAD

**Issue by wcarvalho** [mila-iqia/babyai#105]
I haven't been able to get 99% success rate for PutNextLocal. Are the default command-line arguments correct for that task? I plateau at about 65-70% success rate.

**Comment by maximecb** [comment]
Hello @wcarvalho. Do you mean with reinforcement learning or imitation learning?

The command-line arguments should be as in the paper, but the time required could be as much as a day or so. What frame rate are you getting?

**Comment by wcarvalho** [comment]
I mean with reinforcement learning. I am getting an FPS of 300-600 (but this is with me running 4 experiments per GPU). 
I am using the command-line arguments in the file + code with my own implementation of PPO. I was able to replicate all other results besides PutNextLocall.

Also, the plateau was about 85% and I ran the models for 1e8 steps.

**Comment by maximecb** [comment]
The code was modified since we published the BabyAI paper. There is an [iclr19 branch](https://github.com/mila-iqia/babyai/tree/iclr19) with the code as used in the paper if you want to replicate those results. Otherwise, you might need to tweak the hyper-parameters.

@dyth any comments?

**Comment by dyth** [comment]
@wcarvalho, your FPS seems fine.  See Table 2 in [BabyAI 1.1](https://arxiv.org/pdf/2007.12770.pdf).  Are you using the same model architecture as those described?

**Comment by dyth** [comment]
PutNextLocal is not trainable from pixels with learning rate 10-4.  We found that it worked with 5x10-5 (page 5 of BabyAI 1.1).  You are right, in these experiments we didn't try annealing the learning rate, it would be curious to see whether that stabilises training!

**Comment by wcarvalho** [comment]
This solved it. Had to **remove** annealing the learning rate.

**Comment by maximecb** [comment]
Interesting!
